src/create-config.ts

```diff
@@ -0,0 +1,322 @@
+        id: '${HEARTBEAT_ID}',
+        url: '${HEARTBEAT_URL}',
+      }),
+    },
+    httpGateway: {
+      enabled: true,
+      apiKey: '${HTTP_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    httpSignedDataGateway: {
+      enabled: true,
+      apiKey: '${HTTP_SIGNED_DATA_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    logFormat: 'plain',
+    logLevel: 'INFO',
+    stage: 'dev',
+    skipValidation: true,
+  };
+
+  // Push the NodeSetting secrets to the secrets array
+  secrets.push(`AIRNODE_WALLET_MNEMONIC=`);
+  response.airnodeHeartbeat && secrets.push(`HEARTBEAT_API_KEY=`);
+  response.airnodeHeartbeat && secrets.push(`HEARTBEAT_ID=`);
+  response.airnodeHeartbeat && secrets.push(`HEARTBEAT_URL=`);
+  secrets.push(`HTTP_GATEWAY_API_KEY=`);
+  secrets.push(`HTTP_SIGNED_DATA_GATEWAY_API_KEY=`);
+
+  //generate the triggers from the OISes
+  const oisTriggers: Trigger[] = await Promise.all(
+    Object.keys(apiData.ois).flatMap((oisFilename) => {
+      const ois = apiData.ois[oisFilename];
+      return ois.endpoints.map(async (endpoint) => ({
+        endpointId: await deriveEndpointId(ois.title, endpoint.name),
+        endpointName: endpoint.name,
+        oisTitle: ois.title,
+      }));
+    })
+  );
+
+  const triggers: Triggers = {
+    rrp: oisTriggers,
+    http: oisTriggers,
+    httpSignedData: oisTriggers,
+  };
+
+  const apiCredentials: ApiCredentials[] = Object.keys(apiData.ois).flatMap((oisFilename) => {
+    const ois = apiData.ois[oisFilename];
+    return Object.keys(ois.apiSpecifications.components.securitySchemes).map((security) => {
+      // push the security scheme to the secrets
+      secrets.push(`SS_${security.toUpperCase()}=`.replace(/ /g, '_'));
+      return {
+        oisTitle: ois.title,
+        securitySchemeName: security,
+        securitySchemeValue: `\${SS_${security.toUpperCase()}}`.replace(/ /g, '_'),
+      };
+    });
+  });
+
+  const Config: Config = {
+    chains,
+    nodeSettings,
+    triggers,
+    ois: Object.values(apiData.ois),
+    apiCredentials,
+  };
+
+  //// Build Airkeeper.json ////
+
+  const airkeeperChains: AirkeeperChainConfig[] = apiChains.map((chainName) => {
+    const chainId = chainNameToChainId[chainName];
+    //TODO: Add RrpBeaconServer and DapiServer contracts based on chain
+    const RrpBeaconServer = '';
+    const DapiServer = '';
+
+    return {
+      id: `${chainId}`,
+      contracts: {
+        RrpBeaconServer,
+        DapiServer,
+      },
+    };
+  });
+
+  const airkeeperSubscriptions: Subscriptions = Object.keys(apiData.beacons)
+    .flatMap((beacon) => {
+      const DapiServerInteface = DapiServerInterface();
+      const parameters = '0x';
+      const airnodeAddress = apiData.beacons[beacon].airnodeAddress;
+      const templateId = apiData.beacons[beacon].templateId;
+
+      const threshold = ethers.BigNumber.from(100000000)
+        .mul(apiData.beacons[beacon].updateConditionPercentage * 100)
+        .div(10000);
```

Comment:
I don't know if this is the right way to do this, in santiago's [script](https://github.com/api3dao/airkeeper/blob/a6090d06facc60c16d60b4cbd8f0613c762dc261/scripts/psp-beacon-local-setup.js#L81) he directly divided but I think thats wrong as well.